crypto/dsa/dsa_test.go

```diff
@@ -128,7 +129,7 @@ func TestBytesToPublicKey_BadBytes(t *testing.T) {
 }
 
 func TestBytesToPublicKey_SECP256K1(t *testing.T) {
-	// vector taken from		// vector taken from https://github.com/TBD54566975/web5-js/blob/dids-new-crypto/packages/crypto/tests/fixtures/test-vectors/secp256k1/bytes-to-public-key.json
+	// vector taken from // vector taken from https://github.com/TBD54566975/web5-js/blob/dids-new-crypto/packages/crypto/tests/fixtures/test-vectors/secp256k1/bytes-to-public-key.json
```

Comment:
This comment still looks a bit weird. Is it just a formatting thing? Or is `// vector taken from` duplicated here?

Reply:
lol wot the heck. good catch. fixing

Reply:
donezinos: https://github.com/TBD54566975/web5-go/commit/8e26bcc341c0ed96b93506b478086063600e7854